GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Remotes.cs

```diff
@@ -30,13 +32,19 @@ protected override async Task LoadNodesAsync(CancellationToken token)
                 token.ThrowIfCancellationRequested();
                 var nodes = new Dictionary<string, BaseBranchNode>();
 
+                var gitRemoteManager = new GitRemoteManager(() => Module);
```

Comment:
Actually, this made me realize that the local function CreateRemoteBranchPathNode was capturing gitRemoteManager. I made it a parameter now and left it on this line, since it gets passed in the loop just below it.

Reply:
Decided to put back the declaration where I had it, and have the local function capture it, as before. I think it's the cleanest solution. Coming up with another name for the argument is lame. The only other option is to move the local function to the top of LoadNodesAsync, but that's not awesome either. I think capturing the variable is fine.